modules/i3s/src/i3s-content-loader.ts

```diff
@@ -12,13 +12,15 @@ export const I3SContentLoader: LoaderWithParser = {
   name: 'I3S Content (Indexed Scene Layers)',
   id: 'i3s-content',
   module: 'i3s',
-  worker: true,
```

Comment:
Make the change in 'i3s-content-loader' first and merge into 'master' branch. We won't merge it without worker because it decreases performance very much.

Reply:
Agree